src/app/campaign.model.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @link https://app.swaggerhub.com/apis/Noel/TBG-Campaigns/#/Campaign
+ */
```

Comment:
hmm OK. I'd much rather have it all documented here with docblocks - without looking I don't know whether or not I have a way to edit that documentation. But given that it exists it's good to link to it.

Reply:
Yeah, as I mentioned elsewhere today @bdsl I think the ideal would be to generate it from one source and inline language-specific outputs in all the apps that use it. But I really don't want to start doing this manually since the properties should mean the same thing in all places.

I think for now this gives a decent way to see the model's expected usage in detail. (If we're sticking with it, we probably need the same contract to live on a more affordable multi-user solution than SwaggerHub.)